modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -82,7 +82,23 @@
     /**
      * A mode when data nodes throttle update rate regarding to DR sender load
      */
-    DR_DATA_NODE_SMART_THROTTLING(19);
+    DR_DATA_NODE_SMART_THROTTLING(19),
```

Comment:
FEATURES NOT RESERVED AT:
https://ggsystems.atlassian.net/wiki/spaces/GG/pages/1192198276/Community+edition+features+list